File: StaticSharp/Modifiers/BorderRadius/BorderRadius.js
StaticSharpClass("StaticSharp.BorderRadius", (modifier, element) => {
    StaticSharp.Modifier(modifier, element)

    modifier.Reactive = {
        Radius: undefined,
        RadiusTopLeft: e => e.Radius,
        RadiusTopRight: e => e.Radius,
        RadiusBottomLeft: e => e.Radius,
        RadiusBottomRight: e => e.Radius,
    }


    new Reaction(() => {        
        element.style.borderTopLeftRadius = ToCssSize(modifier.RadiusTopLeft)
        element.style.borderTopRightRadius = ToCssSize(modifier.RadiusTopRight)
        element.style.borderBottomLeftRadius = ToCssSize(modifier.RadiusBottomLeft)
        element.style.borderBottomRightRadius = ToCssSize(modifier.RadiusBottomRight)
    })
})

File: StaticSharp/Modifiers/BorderRadius/BorderRadius.cs
using StaticSharp.Gears;
using System.Runtime.CompilerServices;

namespace StaticSharp {
    public interface JBorderRadius : JModifier {
        public double Radius { get; set; }
        public double RadiusTopLeft { get; set; }
        public double RadiusTopRight { get; set; }
        public double RadiusBottomLeft { get; set; }
        public double RadiusBottomRight { get; set; }
    }


    [ConstructorJs]
    public partial class BorderRadius : Modifier {

    }


}


